src/components/DashboardBody.js

```diff
@@ -49,20 +49,26 @@ const dashboardBodyProps = {
     },
   ]
 }
+const BuildWidgets = widgetArray => {
+
+  return widgetArray.map( widget => {
+    return  {
```

Comment:
Just saw this - single line return, we can remove braces and `return` (but we'll need to wrap the body in parens)